integration-tests/Cargo.toml

```diff
@@ -108,6 +108,7 @@ kusama-runtime = { git = "https://github.com/paritytech/polkadot", branch = "rel
 
 [dev-dependencies]
 xcm-emulator = { git = "https://github.com/zqhxuyuan/xcm-simulator", branch = "v0.9.16" }
+sp-arithmetic = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
```

Comment:
nitpick: missing in the std feature list, or alternatively include without `default-features = false` as it's a dev-dependency